app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DisplayImageSetFragment.kt

```diff
@@ -1,112 +1,134 @@
-package com.github.HumanLearning2021.HumanLearningApp.view.dataset_editing

```

Comment:
This diff is weird because the file was re-encoded from CRLF line terminators to LF, but looking at the actual diff on the command line it looks good to me.